.github/workflows/confrimations-summary.yml

```diff
@@ -0,0 +1,29 @@
+name: Confirmations summary
+on:
+  push:
+    paths:
+      - ui/pages/confirmations/confirm/**
+  pull_request:
+    branches:
+      - develop
+    paths:
+      - ui/pages/confirmations/confirm/**
+
+jobs:
+  comment:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+      contents: write
+    steps:
+      - uses: actions/github-script@v7
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          script: |
+            github.rest.issues.createComment({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: ':warning: Confirmations are currently being redesigned. You have made changes to `ui/pages/confirmations/confirm/` contact Confirmations UX before finalizing your changes. :warning:'
```

Comment:
It would be good to prompt developers to consider wether or not the feature/fix should be added on either the old and/or new confirmations.

Reply:
So we add in the message that they should consider updating the code for old screens since they touched this part of the code?

Reply:
In my view, the value of this notification is pointing out the existence of two parallel confirmation flows, and suggesting that the devs consider updating the new one if they changed the old one only and vice-versa.

The new one exists inside `ui/pages/confirmations/confirm`, the old flows are mainly in `ui/pages/confirmations/*` (except the confirm folder).

The message should be shown on changes to `ui/pages/confirmations` (matching the codeowners file) so that changes to the old flows also trigger it.

Reply:
done